app/backend/src/couchers/models.py

```diff
@@ -189,11 +190,15 @@ class User(Base):
 
     @hybrid_property
     def is_jailed(self):
-        return self.accepted_tos < 1 or self.is_missing_location
+        return (self.accepted_tos < 1) | self.is_missing_location
 
-    @property
+    @hybrid_property
     def is_missing_location(self):
-        return not self.geom or not self.geom_radius
+        return (self.geom == None) | (self.geom_radius == None)
+
+    @hybrid_property
+    def is_visible(self):
+        return ~(self.is_banned | self.is_deleted)
```

Comment:
Doesn't matter so much, but is this clearer as `~is_banned & ~is_deleted`?

Reply:
I think not, but I can see arguments both ways

Reply:
OK, keep as is!